pages/desktop/devhub.py

```diff
@@ -188,3 +197,126 @@ def footer_language_picker(self, value):
     def products_links(self):
         element = self.find_element(*self._footer_products_section_locator)
         return element.find_elements(*self._footer_links_locator)
+
+
+class ConnectFooter(Region):
+    _connect_footer_title_locator = (
+        By.CSS_SELECTOR,
+        '.DevHub-content-header--Connect h2',
+    )
+    _twitter_column_title_locator = (
+        By.CSS_SELECTOR,
+        '.Devhub-content-copy--Connect \
+                                     div:nth-child(1) h4:nth-child(1)',
```

Comment:
This spacing looks weird. Can you double check this?

Reply:
Weird. This should have been corrected when I ran black on the code. I've tried to change the indentation a bit and re-run black. Seems to be looking better now.